package.json

```diff
@@ -50,6 +50,7 @@
   "scripts": {
     "build": "tsc --rootDir ./src",
     "build:watch": "tsc -w --rootDir ./src",
+    "ci": "npm run lint && npm run build && npm test",
```

Comment:
Awesome!
To round this off, can we change Travis and Appveyor to directly use `npm run ci` rather than having the "lint, build, test" logic copied? That way, if we ever change what Travis and Appveyor do, this `ci` script will still be correct

Reply:
Yeah, this sounds like a good idea. This will crash on AppVeyor if I commit this change to this branch since it would try to execute run.sh on a windows server, so I would rather make this change on new feature branch when this and #206 have been merged to master.